www/static/css-src/_docs.scss

```diff
@@ -204,6 +204,23 @@
         }
     }
 
+    .hooks {
+        td[data-col="description"] {
+            text-align: left;
+            vertical-align: middle;
+        }
+
+        td[rowspan="2"], th[colspan="3"], td[data-col="code"] {
+            text-align: center;
+            vertical-align: middle;
+        }
+
+        td[data-col="description"] {
```

Comment:
These should be classes instead of content-specific selectors. Sorry I missed this before this got merged.